index.html

```diff
@@ -11,37 +11,48 @@
     <header class="app-grid__header p-auto bg-hotpink">QUIZ-APP</header>
     <main class="app-grid__scrollarea bg-blue">
       <article class="card position-relative p-20 m-20">
-        <img
-          src="./images/bookmark.png"
-          alt="bookmark"
-          class="position-absolute bookmark"
-        />
+        <button class="position-absolute bookmark">
+          <img
+            src="../../images/bookmark-empty.svg"
+            alt="bookmark"
+            width="28"
+            height="28"
+          />
+        </button>
+
         <h2>QUESTION:</h2>
         <p class="ml-20">
           Lorem ipsum dolor sit amet consectetur, adipisicing elit. Suscipit
           unde vitae numquam distinctio temporibus sapiente eligendi sunt? Sed
           nostrum iste sit quam ullam! Commodi repellendus accusamus sit dolorem
           labore magnam.
         </p>
-        <button class="flex m-auto">SHOW ANSWER</button><br />
+
+        <details class="m-20">
```

Comment:
Nice idea

Reply:
its just for a border between the navigation elements in the bottom menu